src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/UnitPriceType.php

```diff
@@ -26,73 +26,68 @@
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Form\FormInterface;
-use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
 
-/**
- * This base type for shortcut type is used to override the block prefix (which is useful
- * for the form theme rendering) and offer additional options used for rendering as well.
- */
-class ShortcutType extends TranslatorAwareType
+class UnitPriceType extends TranslatorAwareType
 {
     /**
-     * {@inheritDoc}
+     * @var Currency
      */
-    public function buildForm(FormBuilderInterface $builder, array $options)
-    {
-        parent::buildForm($builder, $options);
-        if (empty($options['target_tab']) || empty($options['target_tab_name'])) {
-            return;
-        }
-
-        $options = [
-            'label' => $options['target_tab_name'],
-            'icon' => 'open_in_new',
-            'attr' => [
-                'class' => 'btn btn-link px-0 tab-link',
-                'data-target-tab' => $options['target_tab'],
-            ],
-        ];
-
-        $builder->add('tab_button', IconButtonType::class, $options);
-    }
+    private $defaultCurrency;
 
     /**
-     * {@inheritDoc}
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param Currency $defaultCurrency
      */
-    public function configureOptions(OptionsResolver $resolver)
-    {
-        parent::configureOptions($resolver);
-
-        $resolver->setDefaults([
-            'target_tab' => null,
-            'target_tab_name' => null,
-        ]);
-        $resolver->setAllowedTypes('target_tab', ['string', 'null']);
-        $resolver->setAllowedTypes('target_tab_name', ['string', 'null']);
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        Currency $defaultCurrency
+    ) {
+        parent::__construct($translator, $locales);
+        $this->defaultCurrency = $defaultCurrency;
     }
 
     /**
      * {@inheritDoc}
      */
-    public function buildView(FormView $view, FormInterface $form, array $options)
+    public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        parent::buildView($view, $form, $options);
-        $view->vars['target_tab'] = $options['target_tab'];
-        $view->vars['target_tab_name'] = $options['target_tab_name'];
+        $builder
+            ->add('price', MoneyType::class, [
+                'required' => false,
+                'label' => $this->trans('Retail price per unit (tax excl.)', 'Admin.Catalog.Feature'),
+                'label_help_box' => $this->trans('Indicate the price for one unit of the product. For instance, if you’re selling fabrics, it would be the price per meter.', 'Admin.Catalog.Help'),
```

Comment:
```suggestion
                'label_help_box' => $this->trans('Indicate the price for a single unit of the product. For instance, if you’re selling fabrics, it would be the price per meter.', 'Admin.Catalog.Help'),
```